### PR TITLE
Fix allowing mixed case instance names

### LINKED ISSUE #4618: Unable to upgrade instance to Large 409 name clash error
### Current Behavior

While trying to upgrade the "Main" instance from a Medium to a Large Stack I received the following error:

Screenshot from 2024-10-07 10-15-57

Screenshot from 2024-10-07 10-16-11

### Expected Behavior

The name check should not fail for an existing instance when changing stack type

### Steps To Reproduce

Go to settings -> general on "Main" instance
Select "Change instance Type"
Select "Large" & "4.0.3"

### Environment

- FlowFuse version: 2.9.0
- Node.js version: 18.x
- npm version: 9.x
- Platform/OS: FF Cloud
- Browser: ALL

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

### PR DESCRIPTION
**Pull request by hardillb**
fixes #4618

## Description

Doesn't force existing instance names to lowercase

## Related Issue(s)

#4618 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by hardillb** [comment]
@cstns ready for review now

File: frontend/src/pages/instance/components/InstanceForm.vue [hunk 1]
```diff
@@ -82,7 +82,7 @@
                     <template v-if="creatingNew" #appended-description>
                         <p v-if="hasValidName" class="instance-name-confirmation">
                             <CheckCircleIcon class="ff-btn--icon" />
-                            <span>Your instance will be created as "<i>{{ instanceName }}</i>".</span>
+                            <span>Your instance will be created as "<i>{{ instanceName.toLowerCase() }}</i>".</span>
```
Comment (cstns) [inline frontend/src/pages/instance/components/InstanceForm.vue:85]: If we're accepting instance names with uppercase, is there any point in hiding it from users?

Comment (hardillb) [reply frontend/src/pages/instance/components/InstanceForm.vue:85]:
```suggestion
                            <span>Your instance hostname will be "<i>{{ instanceName.toLowerCase() }}</i>".</span>
```

Comment (hardillb) [reply frontend/src/pages/instance/components/InstanceForm.vue:85]: How about with that change to the label

Comment (cstns) [reply frontend/src/pages/instance/components/InstanceForm.vue:85]: makes more sense!

**Pull request merged**